app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,19 @@
 <resources>
     <string name="app_name">CompareRxJava1VsRxJava2</string>
+    <string name="buffer">Buffer</string>
+    <string name="flatmap">FlatMap</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.flatmap` appears to be unused
 Unused resources make applications larger and slow down builds.